src/utils/cplay.c

```diff
@@ -317,6 +354,9 @@ void play_samples(char *name, unsigned int card, unsigned int device,
 	}
 
 	switch (codec_id) {
+	case SND_AUDIOCODEC_PCM:
```

Comment:
This case should be optional in the code (only if '--enable-pcm=yes' is entered for _configure_ script)...

Reply:
@perexg @vinodkoul thanks for suggestion. Implemented with a new commit. If you think it is necessary I can squash it with previous commit, but I think this way is more clear.

Reply:
hey yes please squash it and make a clean branch for me to merge, thanks

Reply:
@vinodkoul  done